tests/test_posts/test_representative_h_card.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+
+class TestRepresentativeHCard:
+    @pytest.fixture
+    def target(self):
+        from indieweb_utils import get_representative_h_card
+
+        return get_representative_h_card
+
+    def test_representative_h_card(self, target):
+        url = "https://aaronparecki.com"
```

Comment:
This test is making a network call and should `responses` like other tests to prevent network access when running tests.